mappings/net/minecraft/util/math/BlockBox.mapping

```diff
@@ -70,3 +70,5 @@ CLASS net/minecraft/class_3341 net/minecraft/util/math/BlockBox
 	METHOD method_22874 getCenter ()Lnet/minecraft/class_2382;
 		COMMENT @implNote Biased toward the minimum bound corner of the box.
 	METHOD method_27234 infinite ()Lnet/minecraft/class_3341;
+	METHOD method_29299 offset (Lnet/minecraft/class_2382;)V
+		ARG 1 dv
```

Comment:
```suggestion
		ARG 1 delta
```

Reply:
I will keep the argument unmapped for now.

Reply:
what makes delta so bad? maybe offset if you want to name like builders

Reply:
It should be named like `Box.offset(III)` is, and `BlockBox.offset(III)` should as well.

Reply:
just `delta`. `dx` `dy` etc have their `d` stand for "delta", so `delta` is the fitting name.

Reply:
`dx`, `dy`, and `dz` should just be `x`, `y`, and `z` as well. It's much less redundant to say, 'I am offsetting this BlockBox by a Vec3i', rather than, 'I am offsetting this BlockBox by the delta Vec3i'.

Reply:
Would worth a debate in a separate issue.